packages/just-scripts/src/tasks/upgradeRepoTask.ts

```diff
@@ -5,52 +5,55 @@ import {
   findMonoRepoRootPath,
   rushUpdate,
   readRushJson,
-  readPackageJson
+  readPackageJson,
+  logger
 } from 'just-scripts-utils';
 import { getOutdatedStacks } from '../monorepo/getOutdatedStacks';
 import { argv } from 'just-task';
 
-export interface UpgradeRepoTaskOptions {
+interface IUpgradeRepoTaskOptions {
   latest: boolean;
 }
 
 export async function upgradeRepoTask() {
-  const options: UpgradeRepoTaskOptions = {
+  const rootPath = findMonoRepoRootPath();
+  if (!rootPath) {
+    logger.error('Could not find monorepo root path. Not upgrading anything.');
+    return;
+  }
+
+  const scriptsPath = path.join(rootPath, 'scripts');
+  const scriptsPackageJsonPath = path.join(scriptsPath, 'package.json');
+  const scriptsPackageJson = readPackageJson(scriptsPath);
+  if (!scriptsPackageJson) {
+    logger.error(`Could not read ${scriptsPackageJsonPath}. Not upgrading anything.`);
+    return;
+  }
+
+  const options: IUpgradeRepoTaskOptions = {
     latest: argv().latest
   };
 
-  const rootPath = findMonoRepoRootPath();
+  process.chdir(rootPath);
+  const outdatedStacks = await getOutdatedStacks(rootPath);
+
+  outdatedStacks.forEach(stack => {
+    const { dependencies = {}, devDependencies = {} } = scriptsPackageJson;
+    if (devDependencies[stack.name]) {
+      devDependencies[stack.name] = options.latest ? `^${stack.latest}` : `^${stack.wanted}`;
+    } else if (dependencies[stack.name]) {
+      dependencies[stack.name] = options.latest ? `^${stack.latest}` : `^${stack.wanted}`;
+    }
+  });
+
+  fse.writeJsonSync(scriptsPackageJsonPath, scriptsPackageJson, { spaces: 2 });
```

Comment:
I think this way we would potentially write blank objects out for deps and devDeps, right?

Reply:
Yeah, but I don't think that's a problem?